#!/usr/bin/env/python
PACKAGE = "project1"
from dynamic_reconfigure.parameter_generator_catkin import *


gen = ParameterGenerator()

odom_enum = gen.enum([ gen.const("Euler", int_t, 0, "Euler integration method"),
			gen.const("RK", int_t, 1, "Runge-Kutta integration method")],
		 	"Enum of integration methods")


gen.add("odomMethod", int_t, 0, "An integration method parameter which is edited via an enum", 0, 0, 1, edit_method = odom_enum)


exit(gen.generate(PACKAGE, "params", "parameters"))
